astropy/visualization/wcsaxes/coordinate_helpers.py

```diff
@@ -204,10 +209,19 @@ def set_major_formatter(self, formatter):
             raise TypeError("formatter should be a string or a Formatter "
                             "instance")
 
-    def format_coord(self, value):
+    def format_coord(self, value, format='auto'):
         """
         Given the value of a coordinate, will format it according to the
         format of the formatter_locator.
+
+        Parameters
+        ----------
+        value : float
+            The value to format
+        format : { 'auto' | 'ascii' | 'latex' }, optional
```

Comment:
nitpick:  shouldn't this be a comma-separated list (that's what numpydoc uses) instead of pipe-separated?

Reply:
Fixed